app/root.tsx

```diff
@@ -31,16 +31,16 @@ export const links: LinksFunction = () => [
 ];
 
 export async function loader({ request, context }: LoaderArgs) {
-  return json(
-    {
-      user: await getUser(request),
-      effectiveUser: await getEffectiveUser(request),
-      nonce: context.nonce as string,
-    }
-    // ,
-    // { headers: { 'Set-Cookie': await setEffectiveUsername('user3', 'user1') } }
-    // This is how I tested it, after refreshing once since user3 is admin it will mock user1
-  );
+  return json({
+    user: await getUser(request),
+    /* the effectiveUsername, this is for admin impersonating users. Regular
```

Comment:
Can you format this differently?

```ts
user: await getUser(request),

/**
 * effictiveUser: ...
 * ...
 * ...
 */
effectiveUser: await ...
```

Also, `effectiveUser` vs. `effectiveUsername`.